### PR TITLE
add some typings for message api

### PR DESCRIPTION
**Pull request by jin60641**
#278 
It will support Message API's sendSms method

reference
[MessageRequestResponseStatusCode](https://developer.nexmo.com/messaging/sms/guides/troubleshooting-sms)
[MessageRequestResponseSuccess, MessageError, MessageRequestResponse, SendSmsOptions, SendSms, Message](https://developer.nexmo.com/api/sms)

### CODE REVIEW

**Comment by AlexLakatos** [comment]
Messages API is only supported on the beta branch for now, so I'll switch this on the beta branch for now.

**Comment by AlexLakatos** [comment]
@jin60641 this breaks backwards compatibility for Verify. Can you please switch back the Verify responses and add a TODO note to change them when we release a new major version?

**Review by AlexLakatos — CHANGES_REQUESTED** [review]
Switch back the Verify namespacing.

**Comment by jin60641** [comment]
> @jin60641 this breaks backwards compatibility for Verify. Can you please switch back the Verify responses and add a TODO note to change them when we release a new major version?

I switch back the Verify api's namesapcing and add TODO comment on top of file.
Thanks

File: typings/index.d.ts [hunk 1]
```diff
@@ -19,6 +31,99 @@ declare module 'nexmo' {
         parseError: Error;
     }
 
+    /* message API */
+    export enum MessageRequestResponseStatusCode {
```
Comment (Abhi347) [inline typings/index.d.ts:35]: The status codes are returned as string, not integers. You should change the codes to string.

**Comment by Coooi** [comment]
Any updates on this?
I really looking forward to it.

**Comment by kellyjandrews** [comment]
I will evaluate this later this week along with #299 and get these added in.  Thanks for your patience.

File: typings/index.d.ts (hunk 1 above)
Comment (jin60641) [reply typings/index.d.ts:35]: I pushed a new commit. check please.

**Comment by kellyjandrews** [comment]
I pulled these in for now and will start a larger change for v2.9.  Thanks for the contribution.  

If you want to help out with others, I will be creating a new branch `typing-updates` to work towards a more complete set of types.

**Pull request merged**